packages/api/src/server.ts

```diff
@@ -40,28 +42,23 @@ export class Server {
       done();
     });
     // RESPONSE LOG
-    this.app.addHook("preSerialization", (request, reply, payload, done) => {
+    app.addHook("preSerialization", (request, reply, payload, done) => {
       if (payload) {
         request.log.info({ body: payload }, "RESPONSE ::");
       }
       done();
     });
-  }
 
-  setup(): void {
-    this.version();
+    return new Server(app, config);
   }
 
-  version(): void {
-    const { version } = JSON.parse(readFileSync(packageJson).toString());
-    this.app.get("/version", () => {
-      return {
-        version,
-      };
+  setup(): void {
+    this.app.get("*", () => {
+      return "GET requests are not supported. Visit https://docs.etherspot.io/etherspot-skandha-bundler";
```

Comment:
Didnt we look to support both? Or is POST the only recommended?

Reply:
POST is the only option for JSON-RPC services afaik